websites/S/Simkl/metadata.json

```
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.11",
+	"apiVersion": 1,
+	"author": {
+		"id": "354233941550694400",
+		"name": "kuriel."
+	},
+	"service": "Simkl",
+	"description": {
+		"en": "Your All-in-One TV, Anime and Movie Tracking Platform"
```

Official description
```suggestion
		"en": "Track what you watch. Discover new shows and movies. Receive episode notifications. Get personalized recommendations. Join Simkl for free."
```